frontend/src/common/types/card/card.ts

```diff
@@ -0,0 +1,5 @@
+export type CarDataType = {
+  src: string;
+  title: string;
+  price: string;
+};
```

Comment:
Why did you use `type` instead of the interface here?

Reply:
Yes, you are right. In this case, it is better to use the interface. I fixed that